code/datums/supply_packs/ammo.dm

```diff
@@ -44,6 +44,13 @@
 	containername = "\improper .458 bullets crate"
 	group = "Ammo"
 
+/datum/supply_packs/ammo_rounds_box_xm88
+	name = "m252c cartridge crate (x300 rounds)"
```

Comment:
it's called a bullet not a cartridge, cartridge refers to the part of the bullet that holds the powder

this name is misleading